### ISSUE TITLE
Incorrect pin designation for fan connections

### ISSUE THREAD

**Issue by Matakratzka** [Mellow-3D/Fly-SB2040#14]
The pin assignment for PWM0 and PWM1 are reversed on the board. PWM0 (gpio16) belongs to fan1 and PWM1 (gpio17) belongs to fan0.
This should be adjusted in the documentation in the SB2040 cabling section under point 2.6 Fan cabling in the image.
![4-fan](https://github.com/Mellow-3D/Fly-SB2040/assets/68225091/80029cfc-0a8b-45e6-9262-0bd78d570ab0)

**Comment by TheFuzzyGiggler** [comment]
Yeah, I just burned up a perfectly good 4 wire fan because of this. Tried to PWM drive the tach signal because the documentation is poor. I'm seriously thinking about designing my own Toolhead board. Both the BTT and, now it appears, Mellow Fly options have numerous issues and poor design choices. I'm on my 3rd SB2040 board because they've all died for various silly reasons that could have been prevented if the hardware was designed better.
